string header
float64 accX
float64 accY
float64 accZ
float64 GyroX
float64 GyroY
float64 GyroZ
float64 RotX
float64 RotY
float64 RotZ
float64 GavX
float64 GavY
float64 GavZ
float64 LinX
float64 LinY
float64 LinZ
float64 Roll
float64 Pitch
float64 Yaw
float64 X
float64 Y
float64 Z
float64 W
float64 temp1
float64 temp2
